dotnet/src/Functions/Functions.OpenAPI/Extensions/KernelOpenApiPluginExtensions.cs

```diff
@@ -268,9 +268,27 @@ async Task<RestApiOperationResponse> ExecuteAsync(SKContext context)
             })
             .ToList();
 
+        RestApiOperationResponse? restOperationResponse = null;
+        foreach (var response in operation.Responses)
+        {
+            if (response.Key == "200" || response.Key == "2XX" || (response.Key == "default" && restOperationResponse == null))
```

Comment:
Can we check if the key starts with `2` or use a regex to match e.g. `^3\d\d$`
Do we need to worry about the case of "default"?

Reply:
There are options here. The most significant impact here will be on Planning. 

1. Return any response that is 200-299 or 2XX.
2. Return the first response that is 200-299 or 2XX, in that order.
3. Return all responses.

I do believe we need to worry about default - https://spec.openapis.org/oas/latest.html#fixed-fields-13

Reply:
I've refactored things here. Feedback appreciated.